clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/cache/Keys.groovy

```diff
@@ -155,6 +183,18 @@ class Keys {
     "${ID}:${Namespace.LOAD_BALANCERS}:${account}:${region}:${loadBalancerId}"
   }
 
+  static String getVipKey(String vipId, String account, String region) {
+    "${ID}:${Namespace.VIPS}:${account}:${region}:${vipId}"
```

Comment:
From above:

    case Namespace.VIPS.ns:
            if (parts.length == 5) {
               result << [id: parts[2], account: parts[3], region: parts[4]]
      }

When parsing the VIP key, you get id, account, region. When creating the key, you do account, region, id. I don't know how that will work.